tests/sentry/api/endpoints/test_organization_dashboard_details.py

```diff
@@ -203,6 +203,29 @@ def test_response_truncates_with_retention(self):
         assert response.data["expired"]
         assert iso_format(response.data["start"]) == expected_adjusted_retention_start
 
+    def test_release_data_is_returned_when_dashboard_has_release_ids(self):
+        test_release = Release.objects.create(organization_id=self.organization.id, version="v1")
+        filters = {
+            "environment": ["alpha"],
+            "period": "24hr",
+            "release_id": [f"{test_release.id}", "latest"],
+        }
+        dashboard = Dashboard.objects.create(
+            title="Dashboard With Filters",
+            created_by=self.user,
+            organization=self.organization,
+            filters=filters,
+        )
+
+        response = self.do_request("get", self.url(dashboard.id))
+        assert response.data["filters"]["releaseData"] == [
+            {"id": f"{test_release.id}", "version": "v1", "dateCreated": test_release.date_added},
+            {
+                "id": "latest",
+                "version": "latest",
```

Comment:
Should this have `dateCreated` as well? Or is the type null/optional in the frontend?

Reply:
This currently doesn't exist in the frontend so my intention was that this should be optional. There isn't a reasonable `dateCreated` value to pin here because this `latest` option would need to cover multiple projects